libraries/ESP8266SSDP/ESP8266SSDP.cpp

```diff
@@ -439,7 +454,7 @@ void SSDPClass::setDeviceType(const char *deviceType) {
 }
 
 void SSDPClass::setUUID(const char *uuid) {
-  strlcpy(_uuid, uuid, sizeof(_uuid));  
+  snprintf_P(_uuid, sizeof(_uuid), PSTR("%S%s"), F("uuid:"), uuid);  
```

Comment:
or `snprintf_P(_uuid, sizeof(_uuid), PSTR("uuid:%s"), uuid);` ?

Reply:
done ..
I really should start noticing these stuff 😃